### PR TITLE
Multiple Hazard Classification

### PR DESCRIPTION
**Pull request by ismailsunni**
This will replace #3703 
I made different PR to avoid missing review from @Gustry

### CODE REVIEW

**Comment by Gustry** [comment]
Travis is happy 😀

**Comment by ismailsunni** [comment]
Hi @gustry yes it is, but I just thinking about it again. Perhaps I should just create a new method in impact function to update the keywords and checking if the combination is good (e.g. the hazard layer doesn't have classification for the exposure).

File: safe/gui/tools/wizard/step_kw30_field.py [hunk 1]
```diff
@@ -140,6 +129,7 @@ def clear_further_steps(self):
             in order to properly calculate the prev step
         """
         self.parent.step_kw_classify.treeClasses.clear()
+        # self.parent.step_kw_classification.clear()
```
Comment (Gustry) [inline safe/gui/tools/wizard/step_kw30_field.py]: Do you want to keep it ?

File: safe/gui/tools/wizard/step_kw33_multi_classifications.py [hunk 2]
```diff
@@ -0,0 +1,1027 @@
...
+CHOOSE_MODE = 0
+EDIT_MODE = 1
+
+# Min and Max QDoubleEdit
+MIN_VALUE_MODE = 0
+MAX_VALUE_MODE = 1
+
+
+class StepKwMultiClassifications(WizardStep, FORM_CLASS):
+    """Keyword Wizard Step: Multi Classification."""
```
Comment (Gustry) [inline safe/gui/tools/wizard/step_kw33_multi_classifications.py]: One blank line required

File: safe/gui/tools/wizard/step_kw33_multi_classifications.py [hunk 3]
```diff
@@ -0,0 +1,1027 @@
...
+    # noinspection PyMethodMayBeStatic
+    def update_dragged_item_flags(self, item, column):
+        """Fix the drop flag after the item is dropped.
+
+        Check if it looks like an item dragged from QListWidget
+        to QTreeWidget and disable the drop flag.
+        For some reasons the flag is set when dragging.
+
+        :param item:
+        :param column:
```
Comment (Gustry) [inline safe/gui/tools/wizard/step_kw33_multi_classifications.py]: Still empty ?

File: safe/gui/tools/wizard/step_kw33_multi_classifications.py [hunk 4]
```diff
@@ -0,0 +1,1027 @@
...
+
+        .. note:: This is a slot executed when the item change.
+        """
+
```
Comment (Gustry) [inline safe/gui/tools/wizard/step_kw33_multi_classifications.py]: Remove empty line

File: safe/gis/raster/test/test_reclassify.py [hunk 5]
```diff
@@ -18,6 +22,8 @@
 
 
 class TestReclassifyRaster(unittest.TestCase):
+    """Test Reclassify Raster."""
```
Comment (Gustry) [inline safe/gis/raster/test/test_reclassify.py]: Add one line before and after

**Comment by Gustry** [comment]
As you want. If you think it's a a good idea.

**Comment by Gustry** [comment]
Looks good to me, thanks !

**Pull request merged**